engine/Cargo.toml

```diff
@@ -55,4 +48,4 @@ error_refund = ["aurora-engine-precompiles/error_refund"]
 integration-test = ["log"]
 mainnet = ["contract", "log", "aurora-engine-sdk/mainnet"]
 testnet = ["contract", "log", "aurora-engine-sdk/testnet"]
-impl-serde = ["aurora-engine-types/impl-serde", "serde", "aurora-engine-transactions/impl-serde", "evm-core/with-serde"]
```

Comment:
I promise you we cannot remove this `evm-core/with-serde` feature. Try it yourself, if you compile with `impl-serde` it will fail

```
$ cargo check -p aurora-engine --features impl-serde

error[E0277]: the trait bound `ExitError: Serialize` is not satisfied
   --> engine/src/engine.rs:93:14
    |
93  |     EvmError(ExitError),
    |              ^^^^^^^^^ the trait `Serialize` is not implemented for `ExitError`
    |
note: required by a bound in `serialize_newtype_variant`
   --> /home/birchmd/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.137/src/ser/mod.rs:940:12
    |
940 |         T: Serialize;
    |            ^^^^^^^^^ required by this bound in `serialize_newtype_variant`

error[E0277]: the trait bound `ExitFatal: Serialize` is not satisfied
   --> engine/src/engine.rs:95:14
    |
95  |     EvmFatal(ExitFatal),
    |              ^^^^^^^^^ the trait `Serialize` is not implemented for `ExitFatal`
    |
note: required by a bound in `serialize_newtype_variant`
   --> /home/birchmd/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.137/src/ser/mod.rs:940:12
    |
940 |         T: Serialize;
    |            ^^^^^^^^^ required by this bound in `serialize_newtype_variant`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `aurora-engine` due to 2 previous errors
```

Reply:
Fixed with 2bc270a668258512ffafaecc924285649a764d16 by using `evm/with-serde` instead.